src/shared/AuthButton/AuthButton.css

```diff
@@ -0,0 +1,28 @@
+.auth-button {
+    padding: 8px 16px;
+    border: none;
+    border-radius: 4px;
+    font-size: 16px;
+    cursor: pointer;
+    transition: background-color 0.3s ease;
+    margin: 10px;
+  }
+  
+  .blue-button {
+    background-color: #007bff;
+    color: white;
+  }
+  
+  .blue-button:hover {
+    background-color: #0056b3;
+  }
+  
+
```

Comment:
I'm really sorry for nitpicking these tiny formatting issues!!

Reply:
Done and dusted